### ISSUE TITLE
[1.16.5] Error occurred while enabling MobHunting v7.5.7

### ISSUE THREAD

**Issue by AnthonyPaulO** [Teneted/Tenet#1457]
**Minecraft Version :** 1.16.5

**Mohist Version :** mohist-1.16.5-661-server

**Operating System :** Windows 10

**Concerned mod / plugin** : MobHunting v7.5.7
https://www.spigotmc.org/resources/mobhunting.3582/download?version=352447

**Logs :** https://paste.ubuntu.com/p/JcNrdtBbJH/
This is the spigot log for reference (no error here): https://paste.ubuntu.com/p/qvFFPjMpMS/

**Steps to Reproduce :**
1. copy mohist jar to clean folder, create run.bat file to run it (java -jar mohist-1.16.5-661-server.jar)
2. create plugins folder with MobHunting v7.5.7 (https://www.spigotmc.org/resources/mobhunting.3582/download?version=352447), Vault v1.7.3-b131 (https://www.spigotmc.org/resources/vault.34315/download?version=344916), and BagOfGold v3.0.7 (https://www.spigotmc.org/resources/bagofgold.49332/download?version=352446)
3. copy eula file so that you don't get prompted
4. run the bat file

**Description of issue :** 
During initial server run/setup, it produces the following error (which does not occur when running Spigot 1.16.5 server):

[16:35:41] [Server thread/INFO] [/]: [0;33;22m[MobHunting] [mEnabling compatibility with BagOfGold (3.0.8-SNAPSHOT-B490)[m
[16:35:41] [Server thread/INFO] [/]: [0;33;22m[MobHunting] [mMobHunting is using BagOfGold via Vault as Economy Provider[m
[16:35:41] [Server thread/INFO] [/]: [0;33;22m[MobHunting] [mDatabase version 7 detected.[m
[16:35:41] [Server thread/INFO] [/]: [0;33;22m[MobHunting][0;32;1mCopying players from MobHunting til BagOfGoldCore database[m
[16:35:41] [Server thread/ERROR] [Minecraft/]: Error occurred while enabling MobHunting v7.5.7 (Is it up to date?)
java.lang.NullPointerException: null
	at one.lindegaard.MobHunting.storage.SQLiteDataStore.migrateDatabaseLayoutFromV7ToV8(SQLiteDataStore.java:1719) ~[?:?]
	at one.lindegaard.MobHunting.storage.DatabaseDataStore.initialize(DatabaseDataStore.java:279) ~[?:?]
	at one.lindegaard.MobHunting.MobHunting.onEnable(MobHunting.java:193) ~[?:?]
	at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:263) ~[forge:?]
	at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:337) ~[forge:?]
	at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:479) ~[forge:?]
	at org.bukkit.craftbukkit.v1_16_R3.CraftServer.enablePlugin(CraftServer.java:496) ~[forge:7e29f765-296df566-9fb885e8]
	at org.bukkit.craftbukkit.v1_16_R3.CraftServer.enablePlugins(CraftServer.java:410) ~[forge:7e29f765-296df566-9fb885e8]
	at net.minecraft.server.MinecraftServer.func_240787_a_(MinecraftServer.java:432) ~[?:?]
	at net.minecraft.server.MinecraftServer.func_240800_l__(MinecraftServer.java:351) ~[?:?]
	at net.minecraft.server.dedicated.DedicatedServer.func_71197_b(DedicatedServer.java:217) ~[?:?]
	at net.minecraft.server.MinecraftServer.func_240802_v_(MinecraftServer.java:727) ~[?:?]
	at net.minecraft.server.MinecraftServer.func_240783_a_(MinecraftServer.java:271) ~[?:?]
	at java.lang.Thread.run(Thread.java:834) [?:?]

**Comment by Romaindu35** [comment]
This plugin work with spigot server ?

**Comment by AnthonyPaulO** [comment]
> This plugin work with spigot server ?

Yes it does, I mentioned that in the description. Let me know if you have any questions!

**Comment by Romaindu35** [comment]
I have this error with paper server 
![Screenshot_20210623-085433_Chrome](https://user-images.githubusercontent.com/67557705/123050203-bbc0df00-d400-11eb-89a5-dda4b266c143.jpg)

**Comment by AnthonyPaulO** [comment]
> I have this error with paper server

I haven't tried paper, only spigot. I have updated the post and added the spigot log to the log section which shows that there are no errors generated when run from spigot. I will post the spigot log here as well: https://paste.ubuntu.com/p/qvFFPjMpMS/

UPDATE: I just tested against Paper 1.16.5 and I see the error there as well. Spigot 1.16.5 doesn't have the error.
